File: src/main/java/dtos/CreateReleaseRequestDTO.java
package dtos;

public class CreateReleaseRequestDTO {

	private String tagName;
	private String targetCommitish;
	private String name;
	private String body;
	private boolean draft;
	private boolean prerelease;
	private boolean generateReleaseNotes;
	
	public CreateReleaseRequestDTO(String name, String target) {
		this.tagName = name;
		this.targetCommitish = target;
		this.name = name;
		this.body = null;
		this.draft = false;
		this.prerelease = false;
		this.generateReleaseNotes = false;
	}
	
	public String getTagName() {
		return tagName;
	}
	
	public void setTagName(String tagName) {
		this.tagName = tagName;
	}
	
	public String getTargetCommitish() {
		return targetCommitish;
	}
	
	public void setTargetCommitish(String targetCommitish) {
		this.targetCommitish = targetCommitish;
	}
	
	public String getName() {
		return name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	public String getBody() {
		return body;
	}
	
	public void setBody(String body) {
		this.body = body;
	}
	
	public boolean getDraft() {
		return draft;
	}
	
	public void setDraft(boolean draft) {
		this.draft = draft;
	}
	
	public boolean isPrerelease() {
		return prerelease;
	}
	
	public void setPrerelease(boolean prerelease) {
		this.prerelease = prerelease;
	}
	
	public boolean isGenerateReleaseNotes() {
		return generateReleaseNotes;
	}
	
	public void setGenerateReleaseNotes(boolean generateReleaseNotes) {
		this.generateReleaseNotes = generateReleaseNotes;
	}
	
}

File: dtos/CreateReleaseRequestDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GitHubReleasesCLI.NewFolder
{
    public class CreateReleaseRequestDTO
    {
        public string? TagName { get; set; }

        public string? Name { get; set; }

        public string? TargetCommitish { get; set; }

        public string? Body { get; set; }

        public bool Draft { get; set; }

        public bool Prerelease { get; set; }

        public bool GenerateReleaseNotes { get; set; }

        public string? MakeLatest { get; set; }
    }
}
